src/sentry/replays/endpoints/organization_replay_details.py

```diff
@@ -1,26 +1,48 @@
 import uuid
 
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry import features
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.organization import NoProjects, OrganizationEndpoint
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_FORBIDDEN
+from sentry.apidocs.examples.replay_examples import ReplayExamples
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.constants import ALL_ACCESS_PROJECTS
 from sentry.models.organization import Organization
-from sentry.replays.post_process import process_raw_response
+from sentry.replays.post_process import ReplayDetailsResponse, process_raw_response
 from sentry.replays.query import query_replay_instance
+from sentry.replays.validators import ReplayValidator
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class OrganizationReplayDetailsEndpoint(OrganizationEndpoint):
     """
     The same data as ProjectReplayDetails, except no project is required.
     This works as we'll query for this replay_id across all projects in the
     organization that the user has access to.
     """
 
+    public = {"GET"}
+
+    @extend_schema(
+        operation_id="Retrieve a Replay Instance",
+        parameters=[GlobalParams.ORG_SLUG, ReplayValidator],
+        responses={
+            200: inline_sentry_response_serializer("data", ReplayDetailsResponse),
+            400: RESPONSE_BAD_REQUEST,
+            403: RESPONSE_FORBIDDEN,
+        },
+        examples=ReplayExamples.GET_REPLAY_DETAILS,
+    )
     def get(self, request: Request, organization: Organization, replay_id: str) -> Response:
+        """
+        Retrieve a single replay instance.
```

Comment:
More in line with what we have on our docs.
```suggestion
        Return details on an individual replay.
```